tests/realtime/test_AsyncServer.py

```diff
@@ -185,3 +200,132 @@ async def test_boot_a_and_connect_b_and_force_quit_b():
+        for serv in ["scsynth", "supernova"]
+        for opt in [32, 64, 128, 1204, 8192]
+    ],
+)
+async def test_boot_reboot_sticky_options_1(executable, maximum_node_count):
+    server = AsyncServer()
+    port = supriya.osc.utils.find_free_port()
+    options = Options(
+        executable=executable, maximum_node_count=maximum_node_count, port=port
+    )
+    await server.boot(options=options)
+    assert server.is_running
+    assert server._options.maximum_node_count == options.maximum_node_count
+    assert server.port == options.port
+    await server.quit()
+    assert not server.is_running
+    assert server._options.maximum_node_count == options.maximum_node_count
+    assert server.port == options.port
+    await server.boot()
+    assert server.is_running
+    assert server._options.maximum_node_count == options.maximum_node_count
+    assert server.port == options.port
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "executable,memory_size",
+    [(serv, opt) for serv in ["scsynth", "supernova"] for opt in [13131, 21712, 12345]],
+)
+async def test_boot_reboot_sticky_options_2(executable, memory_size):
+    server = AsyncServer()
+    port = supriya.osc.utils.find_free_port()
+    options = Options(executable=executable, memory_size=memory_size, port=port)
+    await server.boot(options=options)
+    assert server.is_running
+    assert server._options.memory_size == options.memory_size
+    assert server.port == options.port
+    await server.quit()
+    assert not server.is_running
+    await server.boot(memory_size=8199)
+    assert server.is_running
+    assert server._options.memory_size == 8199
+    assert server.port == options.port
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "executable,control_bus_channel_count",
+    [(serv, opt) for serv in ["scsynth", "supernova"] for opt in [16385, 6388, 12121]],
+)
+async def test_boot_reboot_sticky_options_3(executable, control_bus_channel_count):
+    server = AsyncServer()
+    port = supriya.osc.utils.find_free_port()
+    options = Options(
+        executable=executable,
+        control_bus_channel_count=control_bus_channel_count,
+        port=port,
+    )
+    await server.boot(options=options)
+    assert server.is_running
+    assert (
+        server._options.control_bus_channel_count == options.control_bus_channel_count
+    )
+    assert server.port == options.port
+    await server.quit()
+    assert not server.is_running
+    await server.boot(options=options, control_bus_channel_count=7777)
+    assert server.is_running
+    assert server._options.control_bus_channel_count == 7777
+    assert server.port == options.port
+    await server.quit()
+    assert not server.is_running
+    await server.boot(options=options)
+    assert server.is_running
+    assert server._options.control_bus_channel_count == control_bus_channel_count
+    assert server.port == options.port
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
```

Comment:
For combinatoric expansion, use two `@pytest.mark.parametrize` decorators, one for each variable you want to parameterize. Pytest will do the expansion for you. Also I'd prefer not having nested list comprehensions around 😆 

For these sticky tests, you can get away with testing the default value for an option and a custom value. Two values is enough to prove that it works, and should cut a lot of time out of the test suite.